rust/kernel/spi.rs

```diff
@@ -199,6 +203,10 @@ impl DriverRegistration {
             .shutdown
             .then(|| DriverRegistration::shutdown_wrapper::<T> as _);
 
+        // SAFETY: Since we are using a `Pin` self, we can register the driver safely as
```

Comment:
`Pin<self>`

Reply:
What I'm trying to say here is that self is Pinned, so I don't think the specification is necessary. What do you think? We could also say `Since we are using a pinned self`

Reply:
Ok but self should be `self`

Reply:
Fixed in 24e91df